src/models/rfcProps.js

```diff
@@ -261,10 +252,6 @@ const properties = {
 		multiple: true,
 		readableName: t('contacts', 'Related'),
 		icon: 'icon-group',
-		info: t(
-			'contacts',
-			'Specify a relationship between another entity and the entity represented by this vCard.'
```

Comment:
It was actually added because lots of people are confused about what some properties really mean. In one case it's easy to understand for the `socialprofile` one, but for the difference between related and relationship, this still needs clarification (gotta love vcard, right?)

Reply:
Do other contact apps show both? Maybe they just change the wording – "Relationship" for one, and some other word for whatever "Related" means. Relationship can have suggestions, could Related as well?

Reply:
> Do other contact apps show both

Yes, they are different properties.

> Maybe they just change the wording

Sure, what do you suggest we use instead? Please remember we do not invent this and this is coming straight from the rfc :)
 

> could Related as well?

Ideally yes, it should suggest other existing contacts for example

Reply:
Fixed this by renaming:
- "Relationship" → "Relationship to you"
- "Related" → "Related contacts"

We can add suggestions in a follow-up.